message/router/middleware/ignore_errors.go

```diff
@@ -20,7 +24,7 @@ func (i IgnoreErrors) Middleware(h message.HandlerFunc) message.HandlerFunc {
 	return func(msg *message.Message) ([]*message.Message, error) {
 		events, err := h(msg)
 		if err != nil {
-			if _, ok := i.ignoredErrors[err.Error()]; ok {
+			if _, ok := i.ignoredErrors[errors.Cause(err).Error()]; ok {
```

Comment:
can you add a test?

Reply:
faa2f17941a31df27f6ec0ca7a5f2a1b91b74203 there you go